src/Operation/Reduction.php

```diff
@@ -26,27 +26,29 @@ final class Reduction extends AbstractOperation
     /**
      * @pure
      *
-     * @return Closure(callable((T|null), T, TKey, Iterator<TKey, T>): (T|null)):Closure (T|null): Closure(Iterator<TKey, T>): Generator<TKey, T>
+     * @template V
+     *
+     * @return Closure(callable(V, T, TKey, Iterator<TKey, T>): V): Closure (V): Closure(Iterator<TKey, T>): Generator<TKey, V>
```

Comment:
can you add a static analysis check for `Reduction` as well? Should be very similar to `ScanLeft`

Reply:
Done!